### PR TITLE
Merge main into release/v5

### PR DESCRIPTION
**Pull request by daniel-almeida-konkconsulting**
# Readiness checklist

-   [X] I added/updated unit tests.
-   [X] I added/updated integration tests.
-   [X] I ensured that the PR title is good enough for the changelog.
-   [X] I labeled the PR.

### CODE REVIEW

File: Modules/Relationships/test/Relationships.Application.Tests/Tests/RelationshipTemplates/Commands/CreateRelationshipTemplate/HandlerTests.cs
Comment (MH321Productions) [inline Modules/Relationships/test/Relationships.Application.Tests/Tests/RelationshipTemplates/Commands/CreateRelationshipTemplate/HandlerTests.cs:1]: You can remove the `Triggers_RelationshipTemplateCreatedDomainEvent` method (and therefore the whole file), as it's not needed anymore

File: Modules/Relationships/test/Relationships.Application.Tests/Tests/Relationships/Commands/CreateRelationship/HandlerTests.cs
Comment (MH321Productions) [inline Modules/Relationships/test/Relationships.Application.Tests/Tests/Relationships/Commands/CreateRelationship/HandlerTests.cs:1]: You can remove the `Publishes_RelationshipStatusChangedDomainEvent` method, as it's not needed anymore.

**Review by MH321Productions — CHANGES_REQUESTED** [review]

File: Modules/Relationships/test/Relationships.Application.Tests/Tests/RelationshipTemplates/Commands/CreateRelationshipTemplate/HandlerTests.cs
Comment (tnotheis) [reply Modules/Relationships/test/Relationships.Application.Tests/Tests/RelationshipTemplates/Commands/CreateRelationshipTemplate/HandlerTests.cs:1]: Confirmed.

File: Modules/Relationships/test/Relationships.Application.Tests/Tests/Relationships/Commands/CreateRelationship/HandlerTests.cs
Comment (tnotheis) [reply Modules/Relationships/test/Relationships.Application.Tests/Tests/Relationships/Commands/CreateRelationship/HandlerTests.cs:1]: Confirmed.

**Review by tnotheis — APPROVED** [review]

**Review by tnotheis — APPROVED** [review]

**Pull request merged**